src/coreclr/debug/daccess/dacdbiimpl.cpp

```diff
@@ -5717,7 +5717,7 @@ void DacDbiInterfaceImpl::GetContext(VMPTR_Thread vmThread, DT_CONTEXT * pContex
         pContextBuffer->ContextFlags = DT_CONTEXT_ALL;
         HRESULT hr = m_pTarget->GetThreadContext(pThread->GetOSThreadId(),
                                                 pContextBuffer->ContextFlags,
-                                                sizeof(*pContextBuffer),
+                                                sizeof(DT_CONTEXT),
```

Comment:
Is this change for clarity? I believe sizeof(*pContextBuffer) is already DT_CONTEXT

Reply:
All of these sizeof(DT_CONTEXT) changes are to make it clearer than we always (even on Windows where the T_CONTEXT/DT_CONTEXT are the same size) pass the size of the original context (DT_CONTEXT).